examples/Cargo.toml

```diff
@@ -0,0 +1,42 @@
+[package]
+name = "zenoh-flow-examples"
+version.workspace = true
+repository.workspace = true
+homepage.workspace = true
+categories.workspace = true
+description.workspace = true
+edition.workspace = true
+license.workspace = true
+readme = "README.md"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[profile.dev]
+debug = true
+
+[profile.release]
+debug = true
```

Comment:
Why do we need these sections?

Reply:
My comment was only addressing the `[profile.dev]` and `[profile.release]` sections.

I don't know where you looked up this information but it's wrong: (i) the `debug = true` controls the amount of debug information included in the compiled binary (see [the official documentation](https://doc.rust-lang.org/cargo/reference/profiles.html#debug)); (ii) compiling with the flag `--release` will still produce artifacts in the `target/release/examples` folder.

Remove the sections.